packages/node/src/indexer/worker/block-dispatcher.service.ts

```diff
@@ -212,17 +212,20 @@ export class BlockDispatcherService
         blockNums,
       );
 
+      const processedBlockCount =
```

Comment:
Getting this every time doesn't make much sense. I think we can keep track of it in memory.

Reply:
Agree, we should only get this once when project initialized